po/de.po

```diff
@@ -1357,7 +1357,7 @@ msgstr[1] "Sollen diese leeren Verzeichnisse entfernt werden?"
 #: ../src/common/film.c:314
 msgid "remove empty directory?"
 msgid_plural "remove empty directories?"
-msgstr[0] "leeres Verzeichnis entfernen?"
+msgstr[0] "Leeres Verzeichnis entfernen?"
 msgstr[1] "leere Verzeichnisse entfernen?"
```

Comment:
Why is the plural (here and the later ones) not capitalized as well?

Reply:
I didn't see that. I will fix it